spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionServiceTests.java

```diff
@@ -193,6 +194,21 @@ public void executeSingleTaskTest() {
 			assertNotNull("TaskDeployment createdOn field should not be null", taskDeployment.getCreatedOn());
 		}
 
+		@Test(expected = InvalidCTRLaunchRequestException.class)
+		@DirtiesContext
+		public void executeSingleTaskTestWithCtrNameSpecified() {
+			initializeSuccessfulRegistry(appRegistry);
+			when(taskLauncher.launch(any())).thenReturn("0");
+			this.taskExecutionService.executeTask(TASK_NAME_ORIG, new HashMap<>(), new LinkedList<>(), "anotherctr");
```

Comment:
It isn't clear to me how the assertions below are going to check the ctr name as I don't see "anotherctr" in those assertions.

Reply:
In this case we are expecting a InvalidCTRLaunchRequestException because the user attempted to apply a CTR app name against a non Composed Task.  So checking for the anotherctr is not necessary.